portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,13 @@
 #content {
   margin-left: auto;
   margin-right: auto;
-  width: 650px;
+  width: 900px;
 }
 
 #greeting-container {
   margin-top: 20px;
 }
+
+#size {
```

Comment:
It is better to use css class instead of use ID to set the css style

<div class="standardSizeBox"></div>


.standardSizeBox {
  width: 500px;
}